SEASONS.md

```diff
@@ -2,29 +2,29 @@
 
 | Season | Start Date | End Date  | DLC Name        | Season of       |
 | :----: | ---------- | --------- | --------------- | --------------- |
-|   1    | 06SEP2017  | 04DEC2017 | Red War         |                 |
-|   2    | 05DEC2017  | 07MAY2018 | Curse of Osiris |                 |
-|   3    | 08MAY2018  | 03SEP2018 | Warmind         | Resurgence      |
-|   4    | 04SEP2018  | 26NOV2018 | Forsaken        | the Outlaw      |
-|   5    | 27NOV2018  | 04MAR2019 | Black Armory    | the Forge       |
-|   6    | 05MAR2019  | 03JUN2019 | Joker's Wild    | the Drifter     |
-|   7    | 04JUN2019  | 30SEP2019 | Penumbra        | Opulence        |
-|   8    | 01OCT2019  | 09DEC2019 | Shadowkeep      | the Undying     |
-|   9    | 10DEC2019  | 09MAR2020 |                 | Dawn            |
-|   10   | 10MAR2020  | 08JUN2020 |                 | the Worthy      |
-|   11   | 09JUN2020  | 09NOV2020 |                 | the Arrival     |
-|   12   | 10NOV2020  | 08FEB2021 | Beyond Light    | the Hunt        |
-|   13   | 09FEB2021  | 10MAY2021 |                 | the Chosen      |
-|   14   | 11MAY2021  | 23AUG2021 |                 | the Splicer     |
-|   15   | 24AUG2021  | 21FEB2022 |                 | the Lost        |
-|   16   | 22FEB2022  | 23MAY2022 | Witch Queen     | the Risen       |
-|   17   | 24MAY2022  | 22AUG2022 |                 | the Haunted     |
-|   18   | 23AUG2022  | 05DEC2022 |                 | Plunder         |
-|   19   | 06DEC2022  | 27FEB2023 |                 | the Seraph      |
-|   20   | 28FEB2023  | 22MAY2023 | Lightfall       | Defiance        |
-|   21   | 23MAY2023  | 21AUG2023 |                 | the Deep        |
-|   22   | 22AUG2023  | 27NOV2023 |                 | the Witch       |
-|   23   | 28NOV2023  | 03JUN2024 |                 | the Wish        |
-|   24   | 04JUN2024  | 07OCT2024 | The Final Shape | Episode: Echoes |
```

Comment:
The script that generates these should probably use UTC...

Reply:
Thanks for that, hard reverted the change but yeah we should probably fix that one